lab2/Chekh_lab-2.py

```diff
@@ -0,0 +1,20 @@
+# Python programming, lab-2, Sasha Chekh
+print('Sasha Chekh')
+
+
+def calculate_math_task(x, y, z):
+    return ((15 - x / y) / pow(33, y) + 19.3) + z
+
+
+def get_number(placeholder, allowed_type=int):
+    input_val = input(f'{placeholder} ')
```

Comment:
you don't need f-string here

Reply:
better to put here the template: 
```suggestion
    input_val = input(f'Please put variable {var_name} for calculation: ')
```
and put variable name as a parameter